CandidateReranking.md

```diff
@@ -0,0 +1,92 @@
+Candidate Reranking Service V1
+==============================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+    - [Top Level Request](#top-level)
+    - [Query Object Parts](#query-object-parts)
+        - [Query](#query)
+        - [Keyword](#keyword)
+        - [Parsed Input](#parsed-input)
+        - [Filter](#filter)
+    - [Profiles](#profiles)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+### Summary
+The Candidate Reranking Service is an HTTP Rest service which performs machine learned ranking on a list of candidate profiles. The service makes use of a versioned reranking-library jar maintained by Texkterkernel which applies a machine learning model trained on Talent Discovery Platform query data.
+
+### Request Structure
+The service supports POST requests to:
+https://api.careerbuilder.com/core/search/candidatereranker
+OAUTH credentials are **required**.
+
+A request is composed of 4 main parts:
+`reranker_config`, `source`, `query`, and `profiles`.
+
+#### Top Level
+| Param    | Type | Description
+|----------|------|--------|
+| reranker_config | String | **Required**: The `reranker_config` determines which learned model to apply to your incoming rerank request. Currently there is only one reranker model to call so while the field is required it can be left empty or filled with a String value like "RerankerV1".
```

Comment:
It's a little odd to have a required field that can be omitted. Also, this looks more like an enum, so it's probably better to be more categorical on the accepted values for the param.

I'd recommend either mandating it to be always supplied with the `"RerankerV1"` value, or not marking it as "Required".

Reply:
also reword 
> a String value like "RerankerV1".
to 
> the String value "RerankerV1".